stages/org.osbuild.rhsm

```diff
@@ -59,6 +58,15 @@ SCHEMA = """
         }
       }
     }
+  }
+},
+"additionalProperties": false,
+"properties": {
+  "dnf-plugins": {
+    "$ref": "#/definitions/plugins"
+  },
+  "yum-plugins": {
+    "$ref": "#/definitions/plugins"
```

Comment:
These should be probably encapsulated in `oneOf` block. However I'm not sure if this will work with subset of object properties...

Reply:
I am using both now actually, since the image has both configuration options, I actually wanna have them in sync. Even though of course if yum is used, configuring dnf makes no sense. But if the file is there, might as well have it say the same thing as the yum one.

Reply:
Fair enough. Also since the stage would produce an error in case e.g. the DNF plugins are being configured, but the configuration paths don't actually exist, I'm probably OK with this.